content/03.use-cases-a.md

```diff
@@ -1,12 +1,14 @@
-## Thirteen practical ways GitHub can accelerate research in ecology and evolution
+## Twelve practical ways GitHub can accelerate research in ecology and evolution
 
 ### Storing and sharing research compendia
 
 <!--*contributors to this section: Dylan Gomes, Emma Hudgins, Pedro Braga, Katherine Hébert* -->
-An EEB research compendium includes all computational materials related to research production, including data, code for analyses and protocols.
+An EEB research compendium includes all computational materials related to research production, including data, code for analyses and protocols. 
 Having copies of these files safely stored is essential to protect against accidental modifications or deletions.
 Many researchers begin using GitHub to store (or backup) their research compendium [@doi:10.1080/00031305.2017.1375986] to a centralized, readily-available remote server (see [Box 1](#definitions)).
 A centralized research compendium stored in a version-controlled repository has the advantages of facilitating collaboration, integrating data and code archiving services, allowing file versions to be accessed and restored, and contributes to open science (see sections below).
+Using version control also allows the research compendium to be used like a virtual lab notebook---when changes are made to files in a version-controlled repository, the author of those changes makes a commit ([Box 1](#definitions)) accompanied by a description of changes.
```

Comment:
This sentence is not very clear to me. I think that the majority of laboratory notebooks serve as "survival guides" for the lab, with instructions on how to access servers, order reagents, code of conduct, etc. I would not say that "traditional" laboratory notebooks are version controlled, which makes me wonder whether this is essential for this part.

Reply:
I have provided a modification below. I will keep leaving these comments as I go, so you can follow.

Reply:
A "laboratory notebook" is usually used to describe the book you keep with detailed notes about your individual experiments in a wet lab or field experiment. This typically includes things like the protocols used and modifications, notes, observations, and results.  I'm trying to distinguish this from a "lab handbook" which would be a resource for the entire lab group and contain code of conduct, onboarding and offboarding procedures, FAQs, how to order reagents, etc.  Maybe "A git repo functions like a digital version of a laboratory notebook because it stores a versioned record of the evolution of a data analysis project with descriptions of changes and additions recorded as commit messages".

Reply:
Please provide suggestions as comments on the PR rather than commits—it makes it easier to discuss and for me to accept, reject, or modify suggestions based on discussion.

Reply:
OK. I will stop providing modifications and just write comments.